pypeclub/OpenPype
# Nuke: multiple baking streams with correct slate

## Brief description
Slate workflow is supporting multiple bake streams

## Description
Since Baking workflow is working with multiple bake look, we needed to make sure slates are reflecting look too in the thumbnail image. 

## Additional info
- thumbnails are also reflecting multiple bake streams
- abstraction of shared codebase for `get_view_process_node` and `duplicate_node`
- removing condition blocking to get input process node in case user had switch of IP function on viewer.

## Testing notes:
1. in settings create two bake presets (withLut, noLut) 
2. make sure `project_settings/nuke/publish/ExtractReviewDataMov/outputs/noLut/bake_viewer_input_process` is OFF
3. open nuke with testing workfile with slate node
4. publish and notice that each of the created bake streams is having correctly looking slate frame
5. !!! make sure you do not have dnxhd in extract review presets tagged as `add_slate` !!! it will fail.

# Depandency
To make this work I have to merge this bugfix: https://github.com/pypeclub/OpenPype/pull/3303
